web/src/App.jsx

```diff
@@ -2,13 +2,18 @@ import { Route, Routes } from "react-router-dom";
 
 import "./App.css";
 import About from "./pages/About.jsx";
+import ConfirmationPage from "./pages/ConfirmationPage.jsx";
 import Home from "./pages/Home.jsx";
 
 function App() {
 	return (
 		<Routes>
 			<Route path="/" element={<Home />} />
 			<Route path="/nested/about/path" element={<About />} />
+			<Route
+				path="/pages/subscribe/confirmation"
```

Comment:
```suggestion
				path="/subscribe/confirmation"
```

You don't need `/pages`. The path shown here is the url path that we want to use in the browser. Eg: http://localhost:3000/subscribe/confirmation